Makefile

```diff
@@ -19,11 +19,15 @@ strip: build
 clean:
 	rm pbapi pbmigrate
 
+
+DOCKER_BUILD_OPTS ?= "--build-arg quay_expiration=2d"
+DOCKER_IMAGE ?= "provisioning-backend"
```

Comment:
Could we move these up the file with a comment for what they are used?
Lukas wants to split up the make file a bit in the future (in the manner Content has it) but in the meantime I'd like to have Vars at the top.